src/components/Chip/Chip.tsx

```diff
@@ -33,8 +33,11 @@ const shakeAnimation = keyframes`
 const StyledChip = styled.button<{ isSelected: boolean; shouldShake: boolean }>`
   flex: 0 0 auto;
   padding: 0.75rem 1.25rem;
-  ${basicSourceSansText};
-  line-height: 100%;
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.1rem;
+  line-height: 1.1rem;
```

Comment:
For me its not that much of an improvement, if we remove a const that defines a base-font, and then we define separately a font for Chip, TextButton, SimpleChip .. etc all separately

Reply:
I already have the next PR in the making 😉